aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -135,6 +135,17 @@ Resources:
           - Effect: Allow
             Action: 'firehose:PutRecord'
             Resource: !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/analysis-events"
+          # S3 access for student-libraries bucket
+          - Effect: Allow
+            Action:
+              - 's3:DeleteObject'
+              - 's3:DeleteObjectVersion'
+              - 's3:GetObject'
+              - 's3:GetObjectVersion'
+              - 's3:PutObject'
+              - 's3:ReplicateObject'
+              - 's3:RestoreObject'
+            Resource: 'arn:aws:s3:::cdo-v3-libraries/libraries_<%=environment%>/*'
```

Comment:
Docs for ^ are here: 
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazons3.html#amazons3-actions-as-permissions
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_s3_rw-bucket.html